### PR TITLE
feat: Add dockerstats quickstart (NR-110988)

### PR DESCRIPTION
**Pull request by gsanchezgavier**
# Summary

Adds a quickstart for the Dockerstats receiver of Opentelemetry collector

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by gsanchezgavier** [comment]
@mangulonr

File: dashboards/docker-otel/docker-otel.json [hunk 1]
```diff
@@ -0,0 +1,890 @@
...
+        {
+          "value": {
+            "string": "sleepy"
+          }
+        },
+        {
+          "value": {
+            "string": "keen_ride"
+          }
+        }
```
Comment (sigilioso) [inline dashboards/docker-otel/docker-otel.json:824]: May we remove these defaults?

Comment (gsanchezgavier) [reply dashboards/docker-otel/docker-otel.json:824]: for sure, good catch.

Comment (gsanchezgavier) [reply dashboards/docker-otel/docker-otel.json]: done

**Review by mangulonr — APPROVED** [review]

**Comment by mangulonr** [comment]
Hi @seemantk

**Review by pkosanam — CHANGES_REQUESTED** [review]
Thank you for these changes and request you to kindly add few alert conditions also.
Doc Ref: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#alerts

File: quickstarts/docker-otel/config.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - New Relic
+installPlans:
+  - docker-otel
+dataSourceIds:
+  - docker-otel
+documentation:
+  - name: Docker (OpenTelemetry) installation docs
+    description: |
+      Docker is a set of platform as a service products that use OS-level
+      virtualization to deliver software in packages called containers.
```
[the remark this answers is not in the document]
Comment (gsanchezgavier) [reply quickstarts/docker-otel/config.yml]: done [2dfd5f4](https://github.com/newrelic/newrelic-quickstarts/pull/1881/commits/2dfd5f433461d104595d6079ddc31e2f8b45c11d)

File: quickstarts/docker-otel/config.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+title: Docker (OpenTelemetry)
+summary: |
+  Monitoring Docker is critical to get instant performance metrics for containerized applications across your entire environment. Install New Relic Docker for Open Telemetry quickstart to proactively instrument Docker with the Dockerstats receiver.
+description: |
+  ## How to monitor Docker with New Relic?
+
+  Dockerstats OpenTelemetry receiver collects metrics from your running Docker containers on the host where OpenTelemetry Collector is installed.
+  The collected metrics allow you to observe the usage of CPU, memory, network, and more.
+
+  A New Relic container Entity will represent each of the containers making it easy to explore, alert, and compare metrics related to them.
```
[the remark this answers is not in the document]
Comment (gsanchezgavier) [reply quickstarts/docker-otel/config.yml]: done [2dfd5f4](https://github.com/newrelic/newrelic-quickstarts/pull/1881/commits/2dfd5f433461d104595d6079ddc31e2f8b45c11d)

File: quickstarts/docker-otel/config.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+  Our monitoring quickstart also contains a Dashboard with the aggregated metrics from all the containers that are reporting to your account with the possibility to apply filters to target the desired workload.
+
+  ### Why monitor Docker with New Relic?
+
+  New Relic's Docker for Open Telemetry quickstart empowers you to get a 360° visibility for your apps, server infrastructure, and Dockerized containers—all in one place. Since your team is likely charged with delivering high-quality software that yields a great customer experience, monitoring Docker is essential. It allows you to keep up that customer experience with your users remaining unaffected by any changes to platforms, tools, languages, or frameworks.
```
[the remark this answers is not in the document]
Comment (gsanchezgavier) [reply quickstarts/docker-otel/config.yml]: done [2dfd5f4](https://github.com/newrelic/newrelic-quickstarts/pull/1881/commits/2dfd5f433461d104595d6079ddc31e2f8b45c11d)

File: quickstarts/docker-otel/config.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+slug: docker-otel
+title: Docker (OpenTelemetry)
+summary: |
+  Monitoring Docker is critical to get instant performance metrics for containerized applications across your entire environment. Install New Relic Docker for Open Telemetry quickstart to proactively instrument Docker with the Dockerstats receiver.
```
[the remark this answers is not in the document]
Comment (gsanchezgavier) [reply quickstarts/docker-otel/config.yml]: done [2dfd5f4](https://github.com/newrelic/newrelic-quickstarts/pull/1881/commits/2dfd5f433461d104595d6079ddc31e2f8b45c11d)

File: install/docker-otel/install.yml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+id: docker-otel
```
[the remark this answers is not in the document]
Comment (gsanchezgavier) [reply install/docker-otel/install.yml]: done [2dfd5f4](https://github.com/newrelic/newrelic-quickstarts/pull/1881/commits/2dfd5f433461d104595d6079ddc31e2f8b45c11d)

File: quickstarts/docker-otel/config.yml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  ### Why monitor Docker with New Relic?
+
+  New Relic's Docker for Open Telemetry quickstart empowers you to get a 360° visibility for your apps, server infrastructure, and Dockerized containers—all in one place. Since your team is likely charged with delivering high-quality software that yields a great customer experience, monitoring Docker is essential. It allows you to keep up that customer experience with your users remaining unaffected by any changes to platforms, tools, languages, or frameworks. 
+
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+installPlans:
+  - docker-otel
```
[the remark this answers is not in the document]
Comment (gsanchezgavier) [reply quickstarts/docker-otel/config.yml]: done [2dfd5f4](https://github.com/newrelic/newrelic-quickstarts/pull/1881/commits/2dfd5f433461d104595d6079ddc31e2f8b45c11d)

**Review by seemantk — APPROVED** [review]
LGTM, thank you for making the changes so quickly

File: alert-policies/docker-otel/MemoryUtilization.yml [hunk 8]
```diff
@@ -0,0 +1,32 @@
+name: Memory Utilization is too high
+
+# Description and details
+description: |
+  This alert is triggered when the Memory Utilization is above 90%.
```
Comment (pkosanam) [inline alert-policies/docker-otel/MemoryUtilization.yml:5]:
```suggestion
  This alert is triggered when the Memory utilization exceeds 90%.
```

File: alert-policies/docker-otel/MemoryUtilization.yml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 90
+    # Time in seconds; 120 - 3600, must be a multiple of 60 for Baseline conditions
+    thresholdDuration: 300
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+
```
Comment (pkosanam) [inline alert-policies/docker-otel/MemoryUtilization.yml:29]:
```suggestion
  - priority: WARNING
    # Operator used to compare against the threshold.
    operator: ABOVE
    # Value that triggers a violation
    threshold: 85
    # Time in seconds; 120 - 3600
    thresholdDuration: 300
    # How many data points must be in violation for the duration
    thresholdOccurrences: ALL
```

**Review by pkosanam — APPROVED** [review]
Looks good to me

**Pull request merged**